models/setup.py

```diff
@@ -0,0 +1,40 @@
+import pip
+import os
+import sys
+import platform
+
+# pip 10.0 introduced a breaking change that moves the location of main
+try:
+    from pip import main
+except ImportError:
+    from pip._internal import main
+
+
+def install_dependencies(packages):
+    pip_arg = ['install'] + packages + ['--no-cache-dir']
+    if hasattr(pip, 'main'):
+        pip.main(pip_arg)
+    else:
+        pip._internal.main(pip_arg)
+
+
+if __name__ == '__main__':
+    install_dependencies(['sklearn', 'pandas', 'numpy',
+                          'textblob', 'nltk', 'scipy'])
+    # Determine if we run python or python3
+    if platform.system() == 'Windows':
```

Comment:
Generally I would advise against doing this; I haven't come across other packages where the application tries to manage this for you. Users typically need to make sure that python is an alias for python3 if they have multiple installations

Reply:
I'm not sure that would work for anaconda or jupyter? We're hoping to make the process as simple as possible for users who choose to "opt-in" since we aren't installing them by default.

Reply:
What distinction are you trying to make between platforms here? This is really only going to cause confusion or do nothing

Reply:
I am not saying this is the right solution we have here (didn't have enough time to carefully look at all the changes) but just want to note that in Anaconda there's no python3 even for Python 3.x environment - there's just python.
As for making users to create aliases for python3... we need to think more about it. We'll need steps for Linux, mac, windows, anaconda, Jupiter and who knows what else. Even now we can see people ignore documented installation steps and after downloading and unpacking zip file trying to be creative for how to configure and start TabPy :)

Reply:
So still not totally clear on the problem being solved here but I would say that by trying to manage this in TabPy you'd be signing up for a lot of non-value added activities. In a very simplistic case, if a user had `python` and `python3` pointing to different environments you'd start executing different interpreters by doing something like this. Even worse, `pip` and `pip3` would probably be two different aliases so you'd be installing packages potentially into a different version of Python than you ultimately want to execute; this will be very confusing to diagnose and help users with.

It's not going to be done in the PR but again just giving a very strong +1 towards providing wheel distributions of TabPy. Requiring a source build and attempting to manually manage environments for people within the package is not something I am aware of any other Python package requiring